tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/BundleUrlValidationService.java

```diff
@@ -38,7 +38,10 @@ protected Status compute()
         }
         catch( Exception e )
         {
-            retval = Status.createErrorStatus( "The bundle URL should be a vaild URL." );
+            if( !bundleUrl.startsWith( "file" ) )
```

Comment:
urls that start with file:// are valid.  So it shoudln't be throwing an exception.  What url did you use that was causing an error?